From cbc2d15720550a782c95b809f6d777ee2452aa16 Mon Sep 17 00:00:00 2001
From: Viktor Hofer <viktor.hofer@microsoft.com>
Date: Thu, 19 Dec 2024 11:18:11 +0100
Subject: [PATCH] Downgrade shared framework versions

Backport: TBD (might not be necessary after the next Arcade update)

---
 eng/Directory.Packages.props | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/eng/Directory.Packages.props b/eng/Directory.Packages.props
index 7114e22690f..934de7ce9de 100644
--- a/eng/Directory.Packages.props
+++ b/eng/Directory.Packages.props
@@ -19,7 +19,9 @@
       We should try to keep this version in sync with the version of app-local runtime in VS.
     -->
     <MicrosoftNetCoreAppRuntimePackagesVersion>8.0.10</MicrosoftNetCoreAppRuntimePackagesVersion>
+    <MicrosoftNetCoreAppRuntimePackagesVersion Condition="'$(DotNetBuildSourceOnly)' == 'true'">8.0.0</MicrosoftNetCoreAppRuntimePackagesVersion>
     <MicrosoftWindowsDesktopAppRuntimePackagesVersion>8.0.10</MicrosoftWindowsDesktopAppRuntimePackagesVersion>
+    <MicrosoftWindowsDesktopAppRuntimePackagesVersion Condition="'$(DotNetBuildSourceOnly)' == 'true'">8.0.0</MicrosoftWindowsDesktopAppRuntimePackagesVersion>
     <_xunitVersion>2.9.2</_xunitVersion>
     <SqliteVersion>2.1.0</SqliteVersion>
   </PropertyGroup>